src/protocols/secure_channel/CASESession.h

```diff
@@ -224,7 +224,9 @@ class DLL_EXPORT CASESession : public Messaging::ExchangeDelegate, public Pairin
     CHIP_ERROR ComputeIPK(const uint16_t sessionID, uint8_t * ipk, size_t ipkLen);
 
     void SendErrorMsg(SigmaErrorType errorCode);
-    void HandleErrorMsg(const System::PacketBufferHandle & msg);
+    CHIP_ERROR HandleErrorMsg(const System::PacketBufferHandle & msg);
```

Comment:
Maybe document that this always returns an error to indicate that the exchange should be closed?